recipes/py2dm/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "py2dm" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/py2dm-{{ version }}.tar.gz
+  sha256: 447e3af1e5adfc7a898acd5e5744ab58f86d995cf615147629fec7b4bf605be7
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 1
```

This should only be incremented for "deployed" builds (i.e. PRs which have been merged into the main branch of a feedstock):
```suggestion
  number: 0
```